module3-crud-and-data-models/r1.1-principles-of-setting-up-your-schema/README.md

```diff
@@ -0,0 +1,57 @@
+## Principles of setting up your schema
+
+Poorly designed databases can cause many problems, including wasting resources, making maintenance difficult, and hindering performance. That's why having a great database schema design is a crucial part of effective data management.
+By defining categories of data and relationships between those categories, database schema design makes data much easier to retrieve, consume, manipulate, and interpret.
+Without a clean, efficient, consistent database schema, you’ll struggle to make the best use of your enterprise data. For example, the same data might be duplicated in multiple locations—or even worse, might be inconsistent between these locations.
+
+### Type of database
+
+Most developers don't see the difference between relational database schema and MongoDB schema, but in reality, it is not the same
+
+<img src="https://i.imgur.com/3yobiPB.jpg" alt="meme" width="400"/>
+
+In a nutshell, in relational databases, models are usually independent of queries, there is no duplication of data as data will _mostly_ be separated into different tables, and it is rigid, you will have to define types and fields for your schema beforehand.
+
+in the other hand, `MongoDB`, you have more flexibility, there are no rules, documents can have new data or fields at any point of time, no need to define types.
+
+### Embedding vs. Referencing
+
+One of the key points to establish a good schema design (especially, if you are using MongoDB) is whether to embed or reference your data.
+
+#### When to `Embed`:
+
+- To retrive all data in a single query.
+- Avoid expense JOINs or $lookups.
+- Update all data with a single operation.
+- In one-to-one or one-to-many relationships between documents.
+
+#### When to `Reference`
+
+- If you need smaller documents or tables, your data will be separated across multiple documents or tables.
+- No duplicate of data.
+- To reduce accessed data not accessed on every query.
+- In many-to-many relationships between tables and documents.
+
+In general, there are few `rules` you can follow to better design your schema:
```

Comment:
same comment as before, please be careful about this, `rules` is not code, so there is no reason to make it in a code (using code backticks for emphasis is not really a thing in any sort of technical writing). also i don't think you need to emphasize this word either way

Reply:
Sorry if this gave you a hard time reviewing, this is my first time writing such a document (also in md), tbh at the beginning I thought backticks are used to emphasize like bold and not only strict for code